PractiTest/src/main/java/api/common/RequestFactory.java

```diff
@@ -0,0 +1,117 @@
+package api.common;
```

Comment:
Please move all files to `com.practitest` namespace. This one for example should be in `com.practitest.api.common` or maybe just `com.practitest.api` package.